example/tools/rustfmt.bzl

```diff
@@ -0,0 +1,30 @@
+def rustfmt_binary_impl(ctx):
+    """Creates an executable target from the current rust toolchain."""
+    toolchain = ctx.toolchains[str(Label("@rules_rust//rust/rustfmt:toolchain_type"))]
+
+    out_file = ctx.actions.declare_file("{name}.rustfmt".format(name = ctx.attr.name))
+
+    script = """#!/usr/bin/env bash
+    {rustfmt} $@
```

Comment:
Maybe this could be more similar to rules_go - would rules_rust accept a PR to expose the rustfmt as an executable target?

Reply:
From https://github.com/bazelbuild/rules_rust/issues/2162 I get the sense it should already be possible?

Reply:
They have a target but it isn't the rustfmt binary itself. It's a rust wrapper with some extra logic.